2015-paypal-intuit/index.html

```diff
@@ -86,13 +91,21 @@ <h1 class="splash">
             <section >
                 <div class="wrapper">
                     <h2 id="topicsLabel">KeyNote Panel</h2>
+                    <p class="abstract">
+                    “How are enterprise organizations winning with NodeJs?”,“What problems are we, as enterprise organizations, trying to solve
+                     with Node”, “What have we learned from the challenges we’ve experience in adopting Node in the enterprise?” – the answers to
+                     these questions are just some of the topics that this panel will discuss as they share their journey with this technology,
+                      born out of open source. The panel is made up of three technology thought leaders: Alex Balazs, Chief Architect at Intuit’s Consumer Group;
+                      Sameera Rao, Director of Engineering, Merchant Engagement at PayPal and Dan Shaw, Co-Founder and President of NodeSource,
+                      the enterprise Node.js company. This panel is moderated by Aliza Carpio, Technology Evangelist at Intuit
+                    </p>
                     <!--<h4 class="well well-sm text-center"><em>More speakers to be announced</em></h4> -->
                 </div>
                 <div class="topic topic-talk">
                     <div class="wrapper">
                         <div class="row">
                             <div class="col-sm-10">
-                                <h3 class="title">Alex Balazs</h3>
+                                <h3 class="title">Dan Shaw</h3>
```

Comment:
Honestly, the only explanation I have here is that I copied everything from `/index.html` to `2015-paypal-intuit/index.html`.